content/en/docs/concepts/services-networking/service.md

```diff
@@ -413,24 +411,25 @@ Kubernetes Service types allow you to specify what kind of Service you want.
 
 The available `type` values and their behaviors are:
 
-[`ClusterIP`](#type-clusterip)
+`ClusterIP`
 : Exposes the Service on a cluster-internal IP. Choosing this value
   makes the Service only reachable from within the cluster. This is the
   default that is used if you don't explicitly specify a `type` for a Service.
-  You can expose the Service to the public internet using an [Ingress](/docs/concepts/services-networking/ingress/) or a
+  You can expose the Service to the public internet using an
+  [Ingress](/docs/concepts/services-networking/ingress/) or a
   [Gateway](https://gateway-api.sigs.k8s.io/).
 
-[`NodePort`](#type-nodeport)
+`NodePort`
 : Exposes the Service on each Node's IP at a static port (the `NodePort`).
   To make the node port available, Kubernetes sets up a cluster IP address,
   the same as if you had requested a Service of `type: ClusterIP`.
 
-[`LoadBalancer`](#loadbalancer)
+`LoadBalancer`
 : Exposes the Service externally using an external load balancer. Kubernetes
   does not directly offer a load balancing component; you must provide one, or
   you can integrate your Kubernetes cluster with a cloud provider.
 
-[`ExternalName`](#externalname)
+`ExternalName`
```

Comment:
These hyperlinks are useful. Why remove them?

Reply:
They are duplicated. You will understand if you search "#externalname" on the page.

Reply:
This is a clickable link to the `#externalname` fragment. It's not a target; it is if you like one of the sources. You click on it and get taken to https://k8s.io/docs/concepts/services-networking/service/#externalname (which is what I intended).

If the UX isn't obvious, we can look to improve it. However I like the idea that you see a summary of the 4 types and, for each type, a page-internal hyperlink lets you click to see more information.

Visit https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types (legacy fragment identifier) to see the 4 summaries and how you can click on each type to get taken to the detailed explanation.

Reply:
I see what you mean now. Will roll back these changes.